src/data-sources/hooks/useDataSources.ts

```diff
@@ -17,6 +17,19 @@ export const useDataSources = < SourceConfig extends DataSourceConfig = DataSour
 		useDispatch< NoticeStoreActions >( noticesStore );
 	const { goToMainScreen } = useSettingsContext();
 
+	const checkDisplayNameConflict = useCallback(
```

Comment:
Nit: `useCallback` isn't useful here, it's not being used as a prop so won't cause rerenders